src/framework/preferences/css/stylus/utils/Themes.styl

```diff
@@ -301,17 +350,44 @@ build-themes-Enactors(contrastThemes) {
                     border-left-color: bColor !important;
                 }
 
+                if (selectedFColor && selectedBColor) {
```

Comment:
Would we want an entry in the `.fl-inverted-color` block for `disabledColor`?

Reply:
My initial thought is that it isn't needed, but I'm open to changing things. If I remember correctly the `fl-inverted-color` class was provided as a means to make certain portions of the content to stick out. This is particularly necessary for our binary colour schemes. In the prefs framework demo we use this for a header. If you were to have a whole section marked as `fl-inverted-color`, which I can't think of a reason for at the moment, I think you'd still want the disabled inputs to look the same for consistency.

Now that i'm thinking more about this, i'm not sure that it's necessary to invert the selection styles either. 

Please let me know what you think about these.